doc/avifdec.1.md

```diff
@@ -49,73 +47,47 @@ Output format can be either JPEG, PNG or YUV4MPEG2 (Y4M).
         - **libgav1**
 
 **-d**, **\--depth** _D_
-:   Output PNG depth.
-    Ignored when the output format is JPEG (always 8 bits per channel) or Y4M
-    (input depth is retained).
-
-    Possible values are:
-
-    :   - **8**
-        - **16**
+:   Output depth [8,16]. (PNG only; For y4m, depth is retained, and JPEG is always 8bpc).
 
 **-q**, **\--quality** _Q_
-:   Output JPEG quality in the range **0**-**100**.
-    Default is 90.
-    Ignored if the output format is not JPEG.
+:   Output quality [0-100]. (JPEG only, default: 90).
 
 **\--png-compress** _L_
-:   Output PNG compression level in the range **0**-**9** (fastest to maximum
-    compression).
-    Default is libpng's built-in default.
-    Ignored if the output format is not PNG.
+:   Set PNG compression level (PNG only; 0-9, 0=none, 9=max). Defaults to libpng's builtin default.
 
 **-u**, **\--upsampling** _U_
-:   Chroma upsampling method.
-    Ignored unless the input format is 4:2:0 or 4:2:2.
-
-    Possible values are:
-
-    :   - **automatic** (default)
-        - **fastest**
-        - **best**
-        - **nearest**
-        - **bilinear**
+:   Chroma upsampling (for 420/422). automatic (default), fastest, best, nearest, or bilinear.
 
 **-r**, **\--raw-color**
-:   Output raw RGB values instead of multiplying by alpha when saving to opaque
-    formats.
-    This is available if the output format is JPEG, and not applicable to y4m.
+:   Output raw RGB values instead of multiplying by alpha when saving to opaque formats
+    (JPEG only; not applicable to y4m).
 
 **\--index** _I_
-:   When decoding an image sequence or progressive image, specify which frame
-    index to decode.
-    Default is 0.
+:   When decoding an image sequence or progressive image, specify which frame index to decode (Default: 0).
 
 **\--progressive**
-:   Enable progressive AVIF processing.
-    If a progressive image is encountered and **\--progressive** is passed,
-    **avifdec** will use **\--index** to choose which layer to decode (in
-    progressive order).
+:   Enable progressive AVIF processing. If a progressive image is encountered and \--progressive is passed,
+    avifdec will use \--index to choose which layer to decode (in progressive order).
 
 **\--no-strict**
 :   Disable strict decoding, which disables strict validation checks and errors.
 
 **-i**, **\--info**
-:   Decode all frames and display all image information instead of saving to
-    disk.
+:   Decode all frames and display all image information instead of saving to disk.
+
+**\--icc** _FILENAME_
+:   Provide an ICC profile payload (implies \--ignore-icc).
```

Comment:
Which style do we choose ? `Provides` or `Provide` ? Below, it is `Specifies`. The usual where `this parameter`is implied?

Reply:
Good point. I removed a bunch of "set" and "provide" "specify" except where I thought it still made sense. There's probably still room for improvement.

Reply:
https://www.gnu.org/prep/standards/html_node/Man-Pages.html#Man-Pages and https://man7.org/linux/man-pages/man7/man-pages.7.html do not seem to address this issue. (I searched for the word "verb" in those two pages and didn't find anything.)

I suggest we follow the convention in the Unix man pages, which seem to use `Provide` rather than `Provides`. Here are three examples:

- cp: https://pubs.opengroup.org/onlinepubs/9799919799/utilities/cp.html
- mv: https://pubs.opengroup.org/onlinepubs/9799919799/utilities/mv.html
- rm: https://pubs.opengroup.org/onlinepubs/9799919799/utilities/rm.html